### PR TITLE
Add extra hp with magic parasyte

### PR DESCRIPTION
**Pull request by Bernatmago**
**How to test**: play the game. get the parasyte in secret area.
**How it works**: Increased hp gets propagated by static variable on level manager, Ilegal code on this pr in general.
**How to go back to normal hp**: when level manager loads lvl 1 ur hp is resetted

### CODE REVIEW

File: Gameplay/entities/player/PlayerController.cpp [hunk 1]
```diff
@@ -1194,47 +1201,47 @@ void Hachiko::Scripting::PlayerController::AttackController()
...
-
-			if (_attack_current_delay > 0.f)
-			{
-				_attack_current_delay -= Time::DeltaTimeScaled();
-				
-				if (_attack_current_delay <= 0.f)
-				{
-					int hit_count = 0;
+//// UNCOMMENT THIS SECTION IF YOU WANT TO DEBUG DRAW ATTACK HIT BOXES
+//if (attack.stats.type == CombatManager::AttackType::CONE)
```
Comment (Bernatmago) [inline Gameplay/entities/player/PlayerController.cpp:1205]: This changed without me touching it?

File: Gameplay/entities/player/PlayerController.cpp [hunk 2]
```diff
@@ -1286,9 +1293,26 @@ void Hachiko::Scripting::PlayerController::CheckNearbyParasytes(const float3& cu
 		}
 		std::vector<GameObject*>& enemies = pack->children;
 
+		float parasyte_pickup_distance = 1.5f;
+
+		if (_magic_parasyte && _magic_parasyte->IsActive())
```
Comment (Bernatmago) [inline Gameplay/entities/player/PlayerController.cpp:1298]: The real change is here, shit code to check for the special parasyte

File: Gameplay/entities/player/PlayerController.cpp (hunk 1 above)
Comment (baransrc) [reply Gameplay/entities/player/PlayerController.cpp:1205]: Probably bc of code formatter.

**Review by baransrc — APPROVED** [review]

**Comment by monikamnc** [comment]
I'm here to say this bad idea it's thanks to me, sorry students I failed and lied you :_

**Pull request merged**